app/code/SlotinDI/CustomWidget/view/frontend/templates/widget/homepage_banner.phtml

```diff
@@ -0,0 +1,10 @@
+<?php if($block->getData('block_id')): ?>
+    <?php echo $block->getLayout()->createBlock('Magento\Cms\Block\Block')->setBlockId($block->getData('block_id'))->toHtml();?>
+<?php endif; ?>
+<?php if($block->getData('title')): ?>
+    <button class="action cms-page-link"
+            onclick="location.href='/geekhub-cms'"
```

Comment:
need use UrlBuilder and `<a>`instead button

Reply:
changed